types.go

```diff
@@ -937,10 +937,19 @@ type (
 	}
 
 	ReferralRequest struct {
-		TrackingID    string      `json:"tracking_id"`
-		Operations    []Operation `json:"operations,omitempty"`
-		Products      []string    `json:"products,omitempty"`
-		LegalConsents []Consent   `json:"legal_consents,omitempty"`
+		TrackingID            string                 `json:"tracking_id"`
+		PartnerConfigOverride *PartnerConfigOverride `json:"partner_config_override,omitemtpy"`
+		Operations            []Operation            `json:"operations,omitempty"`
+		Products              []string               `json:"products,omitempty"`
+		LegalConsents         []Consent              `json:"legal_consents,omitempty"`
+	}
+	
+	PartnerConfigOverride struct {
+		PartnerLogoURL       *string `json:"partner_logo_url,omitempty"`
```

Comment:
i believe for basic types you can do just `string` and `omitempty` will work.

Thats what the rest of the lib does. In order to keep everything in the same way would you be ok changing these to not poiners?